src/engraving/dom/note.cpp

```diff
@@ -3494,4 +3494,70 @@ bool Note::negativeFretUsed() const
 {
     return engravingConfiguration()->negativeFretsAllowed() && m_fret < 0;
 }
+
+bool Note::shouldHideFret() const
+{
+    if (!tieBack() || shouldForceShowFret()) {
+        return false;
+    }
+
+//    if (isContinuationOfBend()) {
+//        return true;
+//    }
```

Comment:
Why not delete commented code?

Reply:
> Why not delete commented code?

to easier keep track if smth goes wrong, I'm picking just some of commits